### DISCUSSION TITLE
Bento Support

### DISCUSSION THREAD

**Discussion by slavakurilyak** [vercel/next.js#37637] [Ideas]
It would be great to see examples of [Bento](https://bentojs.dev/) components (intro [here](https://blog.amp.dev/2021/12/08/introducing-bento/) and [here](https://bentojs.dev/blog/introducing-the-bento-components-library/)).

While there are some bugs remaining ([here](https://github.com/ampproject/bentojs.dev/issues/183) and [here](https://github.com/ampproject/amphtml/issues/38171)), adding Next.js examples of the [Bento components library](https://github.com/ampproject/bentojs.dev) will enable Next team to offer Bento support similar to [AMP Support](https://nextjs.org/docs/advanced-features/amp-support/introduction).

**Comment by 0x15BA88FF** [comment]
I think that's cool... I like the components , they are real easy to work with
